github4s/shared/src/main/scala/github4s/app.scala

```diff
@@ -29,5 +29,6 @@ object app {
   type COGH02[A]   = Coproduct[GistOp, COGH01, A]
   type COGH03[A]   = Coproduct[IssueOp, COGH02, A]
   type COGH04[A]   = Coproduct[AuthOp, COGH03, A]
-  type GitHub4s[A] = Coproduct[GitDataOp, COGH04, A]
+  type COGH05[A]   = Coproduct[GitDataOp, COGH04, A]
+  type GitHub4s[A] = Coproduct[PullRequestOp, COGH05, A]
```

Comment:
Yeah, probably not worthy at this point but it's good to know it for other cases, thanks for sharing!